test/unit/org/apache/cassandra/concurrent/SimulatedExecutorFactory.java

```diff
@@ -153,9 +180,92 @@ public Thread startThread(String name, Runnable runnable, InfiniteLoopExecutor.D
     }
 
     @Override
-    public Interruptible infiniteLoop(String name, Interruptible.Task task, InfiniteLoopExecutor.SimulatorSafe simulatorSafe, InfiniteLoopExecutor.Daemon daemon, InfiniteLoopExecutor.Interrupts interrupts)
+    public Interruptible infiniteLoop(String name,
```

Comment:
I think I added this for journal before switching to mocking it out... Think I can remove to limit the scope of change